### DISCUSSION TITLE
ImageGlass lastest download path

### DISCUSSION THREAD

**Discussion by DerAaron** [d2phap/ImageGlass#1871] [Q&A]
Greetings,

i want to know, if there is a path for "latest download," because i wanna automate updates and upgrades.

With kind regards

Aaron

**Comment by d2phap** [comment]
Hi @DerAaron 

You can use GitHub API to get the latest release: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release


Example: https://api.github.com/repos/d2phap/imageglass/releases/latest
